docs/rules/use-intelligent-tiering.md

```diff
@@ -1 +1,27 @@
-Use Intelligent Tiering
+# S3: use Intelligent-Tiering
+
+The S3 Intelligent-Tiering storage class is designed to optimize storage costs by automatically moving data to the most cost-effective access tier. For a small monthly charge, S3 Intelligent-Tiering monitors access patterns and automatically moves objects that have not been accessed to lower-cost access tiers.
```

Comment:
objects that have not been accessed for a long time ? ou qqhose du genre ?

Reply:
Yes ! Fixed.